lib/view_model/contact_list/contact_list_view_model.dart

```diff
@@ -44,4 +45,20 @@ abstract class ContactListViewModelBase with Store {
   StreamSubscription<BoxEvent>? _subscription;
 
   Future<void> delete(ContactRecord contact) async => contact.original.delete();
+
+  List<ContactRecord> getContacts(bool isEditable, CryptoCurrency? cur) {
+    if (!isEditable) {
+      return contacts.where((element) => element.type == cur).toList();
+    }
+
+    return contacts;
+  }
+
+  List<WalletContact> getWallets(bool isEditable, CryptoCurrency? cur) {
+    if (!isEditable) {
+      return walletContacts.where((element) => element.type == cur).toList();
+    }
+
+    return walletContacts;
+  }
```

Comment:
Also could we make computable trigger to get this automatically and don't call those methods from the widget build and then use those computable getters inside of observers ?

Reply:
computed will only work with getters, so no params can be passed to it, so it's either a function as it is now and contacts will trigger the observer to rebuild thus getting the new contacts
else we can add these 2 params as attributes in the view model and make `getContacts` a computed getter

Reply:
Yep, i meant that make currency and isEditable as part of the view model and then make computed values for contacts and wallets based on that.